docs/_posts/examples/3400-01-31-animate-a-line.html

```diff
@@ -0,0 +1,128 @@
+--- layout: example category: example title: Animate a line description: Animate a line by updating a GeoJSON source on each frame. tags: - layers - sources ---
```

Comment:
this section needs to be formatted like the other examples (not all on one line, etc)